src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1466,22 +1467,57 @@ because xbuild doesn't support framework reference assemblies.
     DependsOnTargets="_LinkAssembliesNoShrinkInputs"
     Condition="'$(PublishTrimmed)' != 'true'"
     Inputs="@(ResolvedAssemblies);$(_AndroidBuildPropertiesCache)"
-    Outputs="@(ResolvedAssemblies->'$(MonoAndroidIntermediateAssemblyDir)%(DestinationSubPath)')">
+    Outputs="@(ResolvedAssemblies->'$(MonoAndroidIntermediateAssemblyDir)%(DestinationSubPath)');@(ResolvedAssemblies->'$(MonoAndroidIntermediateAssemblyDir)%(DestinationSubDirectory)%(Filename).jlo.xml')">
```

Comment:
Do we actually need the two sets of output files here? It seems like the files copied to `$(MonoAndroidIntermediateAssemblyDir)%(DestinationSubPath)` would always change when the `*.jlo.xml` file also changes?

This would make MSBuild check a list of extra filestamps, and maybe it's not needed?

Reply:
I went back and forth on this one. It will absolutely work without the additional check for the normal expected scenarios.  It would fall apart in the "probably would never happen" scenario if someone were to delete the `.jlo.xml` file without deleting the `.dll` file.

If we are ok with this that we can save a few milliseconds by not adding the additional outputs.

Reply:
I wasn't sure either, but I have tested this with incremental builds and they do still build partially.  😄